test-interface/src/main/scala/scala/scalanative/testinterface/NativeRPC.scala

```diff
@@ -16,7 +16,7 @@ private[testinterface] class NativeRPC(clientSocket: Socket) extends RPCCore {
 
   override def send(msg: String): Unit = {
     outStream.writeInt(msg.length)
-    outStream.write(msg.getBytes(StandardCharsets.UTF_16))
+    outStream.write(msg.getBytes(StandardCharsets.UTF_16BE))
```

Comment:
This seems like a good change, but it should probably not be in this PR.

Reply:
Actually without this tests would not start. Test interface expects BigEndian encoding, we can either skip first two bytes in Native side or enforce BE in JVM side. This one seams to be safer. 
Before this fix UTF_16 encoding did not begin with BOM so UTF_16 was equal to UTF_16BE

Reply:
Ah OK, that makes sense.